src/data-comparer.ts

```
@@ -2,35 +2,51 @@
   const comparison: MenuComparison = {};
 
-  // Iterate first over groups, then over items in each group.
-  menus.forEach((menu, menuIndex) => {
-    Object.entries(menu).forEach(([group, items]) => {
-      if (!comparison[group]) {
-        comparison[group] = [];
-      }
+  // Collect all group names across all menus
+  const allGroups = new Set<string>();
+  menus.forEach((menu) => {
+    Object.keys(menu).forEach((group) => allGroups.add(group));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)